### ISSUE TITLE
How to Make My pythonFMU compatible with win32 or x86

### ISSUE THREAD

**Issue by Manish136** [NTNU-IHB/PythonFMU#204]
I have a built a PythonFMU,  it is compatible with win64. I require the same FMU to be compatible with win32. Can explain the procedure.

**Comment by markaren** [comment]
Run `build_xxx.sh` (or otherwise invoke similar commands in a terminal) found in `/pythonfmu/
/pythonfmu-export/`. 

Copy the 32 bit shared library folder from `/resources/` that should appear in `/pythonfmu/resources/binaries/` into the binaries folder inside your FMU.

**Comment by Manish136** [comment]
Thank you for your quick response.
For 32 bit should i change the x64 to x86 ? in build_win.sh file.

**Comment by Manish136** [comment]
It is not working

**Comment by markaren** [comment]
> For 32 bit should i change the x64 to x86 ? in build_win.sh file.

Ah, yes remove `-A x64` for a 32bit build (if this is still the default on MSVC compiler) 

> It is not working

What is not working?

**Comment by Manish136** [comment]
After removing -A x64 still, I'm not getting the any win32 folder under /pythonfmu/resources/binaries/.
 And also under the build folder(which is generated after running the build_win.sh file) , I'm able to see the x64 folder getting generated.

**Comment by Manish136** [comment]
C:/Program Files (x86)/Microsoft Visual Studio/2019/Professional/VC/Tools/MSVC/14.29.30133/bin/Hostx64/x64/cl.exe - skipped

**Comment by Manish136** [comment]
Should i download a CMAKE with 32bit ?

**Comment by markaren** [comment]
> Should i download a CMAKE with 32bit ?

No. Microsoft Visual C++ (MSVC) can compile to both 64 and 32 bit. 32bit is the default, which is why x64 is specified.
Try deleting the /build folder before trying again and also observe the build output for clues.

**Comment by Manish136** [comment]
I tried deleting the build folder, ran the -build_win.sh file, But still not getting the anything under /pythonfmu/resources/binaries/

**Comment by markaren** [comment]
I'm not sure what the issue is. You will need to play around with the settings. You will also need 32-bit Python.

**Comment by Manish136** [comment]
I'm able to build a x86 or Win32 using -DCMAKE_BUILD_TYPE=Release , getting the x86 folder with dll under /pythonfmu/resources/binaries/ . I copied it to the Binaries of the built FMU . Now when i try to Simulate , Simulation is not working .  
Note i have both python-32bit and 64-bit in the ENV path
